### ISSUE TITLE
Hostile bug closing

### ISSUE THREAD

**Issue by jessemyers-lettuce** [PyCQA/flake8#1877]
### how did you install flake8?

```console
n/a
```

### unmodified output of `flake8 --bug-report`

```json
n/a
```

### describe the problem

I field a bug. #1876. @asottile closed and locked the bug without actually reading it.

 > please read the issue template and prompt. this is not the correct place. also
black does not format in f string placeholders so you have unformatted code there

I followed the issue templat to the letter. I read the #filing-a-bug documentation; nothing there suggested that this was not an appropriate place to file a bug.

The bug in this case is a **different behavior between Python 3.11 and 3.12*, not a formatting issue per se. I don't mind if you close the bug and say "this is not something we intend to fix", but please actually read what I wrote.

> also those error codes are correct so don't report them to the correct place either

This doesn't even make sense as English.

But, the point is that these error codes were not reported under Python 3.11 and are under 3.12 and it's not obvious at all why.

**Comment by asottile** [comment]
"problem with E___ or W___ codes
flake8 does not implement any checks, perhaps you want pycodestyle?" and then the bold "NOTE: flake8 is a linting framework and does not implement any checks"

but go off my guy
